lib/brakeman/report/report_text.rb

```diff
@@ -101,6 +102,10 @@ def generate_warnings
     end
   end
 
+  def generate_show_ignored_overview
+    double_space("Ignored File Overview", ignored_warnings.map {|w| output_warning w})
```

Comment:
This should go at the end of the report, after `generate_warnings`, not as part of the summary. It's not a summary, it's a list of warnings.

Reply:
Also title should be "Ignored Warnings" instead of "Ignored File Overview"

Reply:
I see, I went ahead and changed the title here: https://github.com/presidentbeef/brakeman/pull/1861/commits/3203739307155ba34113c958bfee5280412d3389

I moved `generate_show_ignored_overview` to the end below [this line](https://github.com/presidentbeef/brakeman/blob/c83406e7f073ad469cd2f4a44a108cdafd4c6897/lib/brakeman/report/report_text.rb#L23), but I'm getting some test errors so I'll try to push the changes once I figure that part out.

Reply:
Okay, I moved `generate_show_ignored_overview` to the end of the report.

I explicitly return `@output_string` here because adding `generate_show_ignored_overview` at the end was returning `nil` in some cases (When `--show-ignored` isn't used, the `if` statement returns `nil`). All of [these lines](https://github.com/presidentbeef/brakeman/blob/c83406e7f073ad469cd2f4a44a108cdafd4c6897/lib/brakeman/report/report_text.rb#L10-L23)) that use `add_chunk` don't set a second argument so all of the contents are saved to `@output_string`, so I think it should be safe to return the instance variable here explicitly.

Reply:
That makes sense. Just coincidence that it's worked all this time since `generate_warnings` always returns something.